ticketauth/src/main/kotlin/dk/ufst/ticketauth/TicketAuthInitProvider.kt

```diff
@@ -0,0 +1,44 @@
+package dk.ufst.ticketauth
+
+import android.content.ContentProvider
+import android.content.ContentValues
+import android.content.Context
+import android.database.Cursor
+import android.net.Uri
+import android.util.Log
+
+internal var hostApplicationContext: Context? = null
+
+class TicketAuthInitProvider: ContentProvider() {
```

Comment:
Because we already have a `TicketAuth.setup`, I wouldn't mind to pass a `ApplicationContext` into that setup call. Less code in this library and one extra parameter for the client to provide.
But this also works.

Reply:
I agree I may have went a little overboard with that feature given that as you mentioned, we don't explicitly need to setup some stuff before the rest of the app is initialized :).